disco_aws_automation/disco_deploy.py

```diff
@@ -667,12 +667,16 @@ def test(self, dry_run=False, deployment_strategy=None):
         amis = self.get_test_amis()
         if len(amis):
             self.test_ami(random.choice(amis), dry_run, deployment_strategy)
+        else:
+            logging.info("No 'untested' AMIs found.")
 
     def update(self, dry_run=False, deployment_strategy=None):
         '''Updates a single autoscaling group with a newer AMI'''
         amis = self.get_update_amis()
         if len(amis):
             self.update_ami(random.choice(amis), dry_run, deployment_strategy)
+        else:
+            logging.info("No updated AMIs found.")
```

Comment:
The verbage here is a little weird. It might actually be `No outdated AMIs found`. I'm not sure which makes more sense.